development/gcp/cloud-functions/serviceaccountcleaner/README.md

```diff
@@ -0,0 +1,35 @@
+# Cleanup service account secrets using Cloud Function
+
+## Overview
+
+Cloud Function creates a new key for a GCP service account and updates the required secret data. The function is triggered by a  Pub/Sub message sent by a secret stored in Secret Manager.
+
+1. Cloud Scheduler starts the Cloud Function.
```

Comment:
In Overview section you wrote a function is triggered by a pubsub message. Here you wrote it's started by Cloud Scheduler which according to the documentation is a "Fully managed cron job service.". What really triggers this cloud function execution?

Reply:
The function is triggered by a cron job